TekkenBot.py

```diff
@@ -216,8 +216,8 @@
         for channelID in combot_gagged_channels:
             if message.channel.id == channelID:
                 return True
-    else:
-        return False
+    
+    return False
```

Comment:
If the channel id wasn't in the list, this code will fall through and not go through the else, thus not returning 'False', but returning 'None'. Luckily they are mostly equivalent when doing comparisons so it still worked, but i imagine that is not what you intended